src/root/job.tt

```diff
@@ -1,5 +1,5 @@
 [% WRAPPER layout.tt
-    title="Job $project.name:$jobset.name:$job"
+    titleHTML=makeNameLinksForJob(jobset, job)
```

Comment:
Isn't this now missing a text-only title?

Reply:
:+1: fixup'd thanks!